include/openenclave/bits/module.h

```diff
@@ -19,19 +25,65 @@
 
 OE_EXTERNC_BEGIN
 
-/** Load the host file system module. */
+/**
+ * Load the host file system module.
+ *
+ * This function loads the host file system module
+ * which is needed for an enclave application to perform operations
+ * such as mount, fopen, fread and fwrite on files located on the host.
+ *
+ * @retval OE_OK The module was successfully loaded.
+ * @retval OE_FAILURE Module failed to load.
+ *
+ */
 oe_result_t oe_load_module_host_file_system(void);
 
-/** Load the host socket interface module. */
+/**
+ * Load the host socket interface module.
+ *
+ * This function loads the host socket interface module
+ * which is needed for an enclave application to be able to call socket APIs
+ * which are routed through the host.
+ *
+ * @retval OE_OK The module was successfully loaded.
+ * @retval OE_FAILURE Module failed to load.
+ *
+ */
 oe_result_t oe_load_module_host_socket_interface(void);
 
-/** Load the Intel protected file system module. */
+/**
+ * Load the sgx file sytstem module.
+ *
+ * This function enables the use of the SGX file system module.
+ *
+ * @retval OE_OK The module was successfully loaded.
+ * @retval OE_FAILURE Module failed to load.
+ *
+ */
```

Comment:
oe_load_module_sgx_file_system and oe_load_module_host_epoll could be removed as tracked in https://github.com/openenclave/openenclave/issues/1917. But until then, these can be documented in gh pages.